docs/src/Avatar.doc.js

```diff
@@ -127,13 +127,39 @@ card(
     description={`
     If there is no image source provided to the \`Avatar\`, the first character of
     the name provided will be used as a placeholder.
+
+    When \`__dangerouslyUseDefaultIcon\` is set to \`true\`, it will show the default icon
```

Comment:
@nachofassini removed the doc update, since we don't want anyone outside of pinterest to use this temporary prop